ckan_meta_tester/__init__.py

```diff
@@ -10,10 +14,27 @@ def test_metadata() -> None:
     logging.getLogger('').setLevel(
         environ.get('INPUT_LOG_LEVEL', 'info').upper())
 
+    github_token = environ.get('GITHUB_TOKEN')
+
     ex = CkanMetaTester(environ.get('GITHUB_ACTOR') == 'netkan-bot')
     exit(ExitStatus.success
-         if ex.test_metadata(environ.get('INPUT_SOURCE',            'netkans'),
-                             environ.get('INPUT_PULL_REQUEST_BODY', ''),
-                             environ.get('GITHUB_TOKEN'),
+         if ex.test_metadata(environ.get('INPUT_SOURCE', 'netkans'),
+                             get_pr_body(github_token, environ.get('INPUT_PULL_REQUEST_URL')),
+                             github_token,
                              environ.get('INPUT_DIFF_META_ROOT'))
          else ExitStatus.failure)
+
+
+def get_pr_body(github_token: Optional[str], pr_url: Optional[str]) -> str:
+    # Get PR body text
+    if pr_url:
+        headers = { 'Accept': 'application/vnd.github.v3.raw+json' }
+        parsed_pr_url = urlparse(pr_url)
+        if github_token:
+            if parsed_pr_url.scheme == 'https' and parsed_pr_url.netloc == 'api.github.com' \
+                    and parsed_pr_url.path.startswith('/repos/'):
+                headers['Authorization'] = f'Bearer {github_token}'
```

Comment:
I thought the header used `token`, not `Bearer`, and the docs seem to bear :bear: this out:

https://docs.github.com/en/rest/guides/getting-started-with-the-rest-api

Reply:
From the link you posted earlier that specifically explains how to use the GITHUB_TOKEN in a workflow:
https://docs.github.com/en/actions/reference/authentication-in-a-workflow#using-the-github_token-in-a-workflow

![image](https://user-images.githubusercontent.com/28812678/123520064-1d0dda00-d6af-11eb-857c-cb2618464f2e.png)

Another case of the GitHub API being inconsistent? Or are those two actually different?

Reply:
Well the netkan.exe calls are already using `Authorization: token` with these tokens, so that should at least be safe.

Reply:
Changed to token then, and added a warning log message in case the request fails (don't want to fail the whole validation, the PR body is not necessarily required in the most common case), e.g. due to authorization failure.
We'll know sooner or later if that's the correct one. Another PR we can prepare for the docs.